lib/models/user.dart

```diff
@@ -0,0 +1,21 @@
+class User {
+  final int uid;
+  final String firstname;
+  final String lastname;
+  final String email;
+  final String password;
```

Comment:
does password need to be stored in user class ? can this info be leaked ?

Reply:
copied these from api will change soon